Packs/SecurityTrails/Integrations/SecurityTrails/SecurityTrails.py

```diff
@@ -0,0 +1,1069 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+''' CONSTANTS '''
+
+removed_keys = ['endpoint', 'domain', 'hostname']
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def handle_error(self, message: dict = {}):
+        return_error(message['message'])
+
+    def domain_tags(self, hostname: str = None):
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}/tags',
+            ok_codes=[200, 403],
+            resp_type="response"
+        )
+        if res.status_code != 200:
+            self.handle_error(res.json())
+        else:
+            return res.json().get('tags', [])
+
+    def domain_details(self, hostname: str = None):
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}',
+            ok_codes=[200, 403],
+            resp_type="response"
+        )
+        if res.status_code != 200:
+            self.handle_error(res.json())
+        else:
+            return res.json()
+
+    def domain_subdomains(self, hostname: str = None, children_only: str = 'true'):
+        query_string = {"children_only": children_only}
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}/subdomains',
+            params=query_string,
+            ok_codes=[200, 403],
+            resp_type="response"
+        )
+        if res.status_code != 200:
+            self.handle_error(res.json())
+        else:
+            return res.json()
+
+    def associated_domains(self, hostname: str = None, page: int = 1):
+        params = {
+            "page": page
+        }
+        res = self._http_request(
+            'GET',
+            f'domain/{hostname}/associated',
+            params=params,
+            ok_codes=[200, 403],
+            resp_type="response"
+        )
+        if res.status_code != 200:
+            self.handle_error(res.json())
+        else:
+            return res.json()
+
+    def get_ssl_certificates(self, query_type: str = "stream", hostname: str = None, params: dict = {}):
+        if query_type == "paged":
+            res = self._http_request(
+                'GET',
+                f'domain/{hostname}/ssl',
+                params=params,
+                ok_codes=[200, 403],
+                resp_type="response"
+            )
+        elif query_type == "stream":
+            res = self._http_request(
+                'GET',
+                f'domain/{hostname}/ssl_stream',
+                params=params,
+                ok_codes=[200, 403],
+                resp_type="response"
+            )
+        if res.status_code != 200:
+            self.handle_error(res.json())
+        else:
+            text = res.text
+            text = f"[{text}]"
+            text = text.replace("}{", "},{")
```

Comment:
what is the purpose of it?

Reply:
I had forgotten about this. This is actually a bug in their API. Their JSON return syntax is incorrect. I had forgotten to remove it. L87-90 can be removed and L91 indented to just return the res.json()